extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaInfoboxInsertDialog.js

```diff
@@ -243,8 +246,44 @@ ve.ui.WikiaInfoboxInsertDialog.prototype.createEmptyStateContent = function ( un
  * @desc sets HTML content for insert infobox dialog
  * @param {HTMLElement} $content
  */
-ve.ui.WikiaInfoboxInsertDialog.prototype.setDialogContent = function ( $content ) {
-	this.$body.html( $content );
+ve.ui.WikiaInfoboxInsertDialog.prototype.setDialogContent = function ($content) {
+	this.$body.html($content);
+
+	if (window.wgEnablePortableInfoboxBuilderInVE) {
+		this.$body.append(this.addNewTemplateWidget());
+	}
+};
+
+ve.ui.WikiaInfoboxInsertDialog.prototype.refreshInfoboxesList = function () {
+	this.getInfoboxTemplates()
+		.then(this.createDialogContent.bind(this))
+		.then(this.setDialogContent.bind(this));
+};
+
+ve.ui.WikiaInfoboxInsertDialog.prototype.addNewTemplateWidget = function () {
+	var $container = $('<div>', { class: 've-ui-wikiaInfoboxInsertDialog-add-container' }),
```

Comment:
Haven't found any that simple.
https://www.mediawiki.org/wiki/OOjs_UI
Most of components are more complex and we don't need them here.